playbooks/roles/hammer_devel/tasks/ruby-2-3-1.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: 'Remove Old Ruby'
+  become: true
+  yum: name=ruby,ruby-libs state=absent
+- name: 'Install Ruby'
+  become: true
+  yum: name=https://github.com/feedforce/ruby-rpm/releases/download/2.3.1/ruby-2.3.1-1.el7.centos.x86_64.rpm state=present
```

Comment:
If this is applied to an existing environment this could have unintended consequences for the developer. I get how this makes things easier on a stand-alone deployment. I just wonder about invasive changes like this vs using RVM or even trying to use software collections if RPMs is the goal [1].

[1] https://www.softwarecollections.org/en/scls/rhscl/rh-ruby23/